build.bat

```diff
@@ -252,7 +252,7 @@ REM USAGE function: output a usage message
 	echo The following arguments are supported:
 	echo    help^|*?*    : output this message and exit; ?? for more options.
 	echo    32^|64       : set the architecture to x86 or x64, respectively;
-	echo                   if none is specified, autodetection is attempted.
+	echo                  if none is specified, autodetection is attempted.
```

Comment:
nit: If you're changing the alignment so all the text starts in the same column then maybe the two lines above should have one space removed too?

Reply:
Thanks.
The explanations are now indeed all aligned. The first two lines contain the `^` `cmd` escape char, so they have in fact one char less printed (which I've failed to notice before).

```
> build.bat ??
build.bat: using CMAKE binary: C:\Progra~1\CMake\bin\cmake.exe : cmake version 3.12.0
Usage: C:\Users\bpi\RnD\elastic\repos\bpintea\elasticsearch-sql-odbc\build.bat [argument(s)]

The following arguments are supported:
   help|*?*    : output this message and exit; ?? for more options.
   32|64       : set the architecture to x86 or x64, respectively;
                 if none is specified, autodetection is attempted.
   setup       : invoke MSVC's build environment setup script before
                 building (requires 2017 version or later).
   clean       : remove all the files in the build dir.
   proper      : clean libs, builds, project dirs and exit.
   type:T      : selects the build type, T: Debug or Release.
   ctests      : run the CI testing - integration and unit tests,
                 x86 and x64 architectures in Release and Debug mode
                 - and exits. (All the other arguments are ignored.)
   itests[:I]  : run the integration tests. If I, the path to the
                 packaged driver (installer) is not provided, this
                 will be looked for in the installer output folder.
   utests      : run all the defined unit tests.
   suites      : run all the defined unit tests, individually.
   suite:U     : run one unit test, U.
   package[:V] : generate the installer. V is a versioning string
                 that will be added to the installer file name and can
                 can only be specified before the project/make files
                 are generated.
   sign[:C+P]  : sign the installer file. C is the certificate file to
                 use and P the file containing the password for it (if
                 any).

Installing and packaging are mutually exclusive.
Multiple arguments can be used concurrently. Examples:
- set up a x86 build environment and running the unit tests:
  > C:\Users\bpi\RnD\elastic\repos\bpintea\elasticsearch-sql-odbc\build.bat setup 32 utests
- remove any existing build and create a release archieve:
  > C:\Users\bpi\RnD\elastic\repos\bpintea\elasticsearch-sql-odbc\build.bat clean package:-final type:Release

List of read environment variables:
   CMAKE       : cmake executable to use (if not in PATH);
                 currently set to: `C:\Progra~1\CMake\bin\cmake.exe`.
   BUILDS_DIR  : folder path to build the driver into;
                 currently set to: `C:\Users\bpi\RnD\elastic\repos\bpintea\elasticsearch-sql-odbc\\builds`.
   INSTALL_DIR : folder path to install the driver into;
                 currently set to: ``.

Extra development arguments:
   nobuild     : skip project building (the default is to build).
   genonly     : generate project/make files, but don't build.
   curldll     : link libcurl dynamically.
   exports     : dump the exported symbols in the DLL after building.
   depends     : dump the dependents libs of the build DLL.
   install[:D] : install the driver files. D, the target directory
                 path can only be specified before the project/make
                 files are generated. (This does not run the
                 installer.)
   regadd      : register the driver into the registry unde
                 'Elasticsearch ODBC' name;
                 (needs Administrator privileges).
   regdel      : deregister the driver from the registry;
                 (needs Administrator privileges).
   tests       : (deprecated) synonym with utests.
```